internal/claude/parser.go

```
@@ -398,10 +435,29 @@ func findParentRepository(directory string) string {
+				return repoName
+			}
+		} else {
+			// Try to get repository name from parent directory
+			repoName := git.GetRepositoryName(parentDir)
+			// Cache the parent result (only if not empty)
+			if repoName != "" {
+				repositoryCache[parentDir] = repoName
+				// Cache this directory's result too
+				repositoryCache[directory] = repoName
+				return repoName
+			} else {
```

golangci-lint (1.64.8)

[warning] indent-error-flow: if block ends with a return statement, so drop this else and outdent its block

(revive)